.editorconfig

```diff
@@ -21,7 +21,3 @@ root = true
 charset = utf-8
 end_of_line = lf
 insert_final_newline = true
-
-[*.{java,xml,py}]
-indent_style = space
-indent_size = 4
```

Comment:
not sure why delete this code?

Reply:
The reason for this change is that we encountered a problem while formatting the java code,

The intended effect of format is to automatically wrap and indent a line when it exceeds the maximum length of 100, which is a special case where the number of indented Spaces is eight

If the changes in the.editorConfig do not change, the indentation will change to 4, even though the code style has been set to 8

## Before modification  

![img_v3_028c_a7aac239-0ec9-4b39-bc7f-3aa42f6f1cag](https://github.com/apache/incubator-hugegraph/assets/103014488/d39d6c16-ca19-4421-89b2-c58f88cc06d7)

## After modification

![img_v3_028c_492f8f97-bc13-47a0-ba66-b4f675c0851g](https://github.com/apache/incubator-hugegraph/assets/103014488/b8905db3-b69f-4a99-acfa-b774d9488418)

Reply:
I'm sorry, but I've found a new solution that doesn't require deleting the above, but instead enables a special option in`.editorconfig` for indenting more than the maximum line wrap of a single character

I'll make a new commit, which will undo my changes to the`.editorconfig` section and add the options above